web-app/package.json

```diff
@@ -3,20 +3,22 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@emotion/react": "^11.11.1",
-    "@emotion/styled": "^11.11.0",
+    "@emotion/react": "^11.13.0",
+    "@emotion/styled": "^11.13.0",
     "@fortawesome/fontawesome-svg-core": "^6.4.2",
     "@fortawesome/free-brands-svg-icons": "^6.4.2",
     "@fortawesome/free-regular-svg-icons": "^6.4.2",
     "@fortawesome/free-solid-svg-icons": "^6.4.2",
     "@fortawesome/react-fontawesome": "^0.2.0",
-    "@mui/icons-material": "^5.14.9",
-    "@mui/material": "^5.14.9",
+    "@mui/icons-material": "^5.16.4",
+    "@mui/material": "^5.16.4",
+    "@mui/x-date-pickers": "^7.11.0",
     "@mui/x-tree-view": "^6.17.0",
     "@reduxjs/toolkit": "^1.9.6",
     "@turf/center": "^6.5.0",
     "@types/i18next": "^13.0.0",
     "@types/leaflet": "^1.9.12",
+    "axios": "^1.7.2",
```

Comment:
I don't see an instance where this package is used, would it be safe to remove it?

Reply:
certainly -- i'll remove it as part of the the GBFS ticket + GTFS minor updates!